Formula/g/glab.rb

```diff
@@ -18,9 +19,7 @@ class Glab < Formula
   depends_on "go" => :build
 
   def install
-    ENV["CGO_ENABLED"] = "1" if OS.mac?
```

Comment:
This is generally used to reuse existing binaries present on macOS

The used upstream `Makefile` https://gitlab.com/gitlab-org/cli/-/blob/v1.55.0/Makefile#L7 allows that via:
`export CGO_ENABLED ?= 0`

If there is no specific reason to prevent reusing libraries on macOS i would suggest to put that line back.